public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataAlertingTab.test.tsx

```diff
@@ -316,25 +337,6 @@ describe('PanelAlertTabContent', () => {
     };
 
     expect(defaultsWithDeterministicTime).toMatchSnapshot();
-
```

Comment:
we agreed on removing this test after talking to @konrad147 as it's not clear that is testing the right thing.
I tested manually, and the link to the alert rule is correctly updated once the variable changes